KachnaOnline.Business/Services/EventsService.cs

```diff
@@ -0,0 +1,239 @@
+// EventsService.cs
+// Author: David Chocholatý
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoMapper;
+using KachnaOnline.Business.Configuration;
+using KachnaOnline.Business.Constants;
+using KachnaOnline.Business.Data.Repositories.Abstractions;
+using KachnaOnline.Business.Exceptions;
+using KachnaOnline.Business.Exceptions.Events;
+using KachnaOnline.Business.Extensions;
+using KachnaOnline.Business.Services.Abstractions;
+using KachnaOnline.Business.Models.Events;
+using Microsoft.Extensions.Logging;
+using EventEntity = KachnaOnline.Data.Entities.Events.Event;
+
+namespace KachnaOnline.Business.Services
+{
+    public class EventsService : IEventsService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IMapper _mapper;
+        private readonly ILogger<EventsService> _logger;
+        private readonly IEventsRepository _eventsRepository;
+        private readonly IUserRepository _userRepository;
+
+        private const int EventPlanningEnterTimeout = 1000;
```

Comment:
I'd probably move this to `Business.Constants`

Reply:
Good thinking. I will do that. It is mostly pre-preparation for linking states to events, but for now it can be moved there.